Apigee/SmartDocs/Revision.php

```diff
@@ -301,12 +306,15 @@ public function toArray($verbose = TRUE)
      *
      * @param OrgConfig $config
      * @param string $modelId
+     * @param string $modelName
      */
-    public function __construct(OrgConfig $config, $modelId)
+    public function __construct(OrgConfig $config, $modelId, $modelName = "")
```

Comment:
UUID can be used to get modelName and is a great unique identifier (hence the name), so why pass both?